packages/atomic/src/components/facets-v1/facet-value-checkbox/facet-value-checkbox.tsx

```diff
@@ -31,17 +31,9 @@ export const FacetValueCheckbox: FunctionalComponent<FacetValueProps> = (
       <label
         htmlFor={id}
         part="value-checkbox-label"
-        class="w-full flex items-baseline pl-2 py-2.5 text-on-background cursor-pointer ellipsed"
+        class="w-full flex items-center pl-2 py-2.5 text-on-background cursor-pointer ellipsed"
```

Comment:
I remember having to set items-baseline because of a CSS issue with some font https://github.com/coveo/ui-kit/pull/923 Why do you need to set it back to center?

Reply:
With the icons, there was a slight alignment issue with the span responsible for the count and any other type of text added next to them. I used this to center everything together. Maybe a vertical align middle on the text could work? I will try it.

Reply:
With `items-baseline`:
![Screen Shot 2021-07-22 at 12 12 54 PM](https://user-images.githubusercontent.com/83350303/126672613-c18f52e4-45b7-440b-bc2c-d3e7538e6a90.png)

Even with tailwind `align-middle` class on the count span, it still does not align nicely. Adding margins or padding to either the icons or the text also doesn't help. `items center` is the only way I've been able to get them to align correctly.

Reply:
Fair enough!